### ISSUE TITLE
[Bug] Tab key problem in justified text mode

### ISSUE THREAD

**Issue by koch17** [cryptee/web-client#48]
Hi John,

Me again. When the text is left aligned, the tab key does what it should, but when text is justified, the tab key doesn't react no matter how many times you press it, but if you switch to left aligned again, all the times you pressed tab key appear. Just check the gif. Also, after you have pressed the tab key, when you switch to justify, it pushes text forward again, hiding the space you have created by the tab key.
This behavior is similar both in firefox and chromium.

![crypt](https://user-images.githubusercontent.com/50534491/77492175-3c67e880-6e16-11ea-9d8d-efe4a65ae041.gif)

**Comment by johnozbay** [comment]
Hi there! Thanks a lot for this! 

My bad! I've finally figured out what caused this. Looks like while fixing the spacebar, I broke this :

https://github.com/cryptee/web-client/blob/cf2ae5f0f10714834cc683c2128a08566a06a1bd/source/css/base.css#L808-L810

Looks like this needs to be `pre-wrap` instead of `normal` for tabs to work. (But this breaks the way Firefox displays justified text altogether) 

Will figure out a solution, and push out a fix with the upcoming bug fixes release.

**Comment by johnozbay** [comment]
Just a quick update – 

Still looking into how this can be fixed this without having to make browser/vendor-specific prefixes for maximum compatibility. Looks like it's a deeply rooted issue with the editor framework Cryptee's using, so will need to do some further changes to be able to fix this without breaking something else. 

Thank you for your understanding and patience in the meantime ✌🏻

**Comment by koch17** [comment]
No problem at all. It doesn't really bother me, just want cryptee to be at its best.

**Comment by johnozbay** [comment]
Hi there! 

We've shipped a fix for this in v3, and it sort-of works around Firefox's quirks by faking tab-clicks in the keyboard. It can break words, but doesn't work between sentences. We've normalized it as much as possible here : 

https://github.com/cryptee/web-client/blob/3404467b740fbe4a407585bfd715f33e2a4aa259/source/js/quill-modules/init.js#L34

And in the upcoming release, we're shipping a little patch we forgot to apply in CSS that should fix 95% of the use cases. 
I've linked this issue to our internal issue/release tracker. Once the update with this CSS fix is out (should be today / tomorrow), this issue will be closed automatically, and you'll get a notification!

Thank you so much for your help and patience! ✌🏻  

Deeply appreciate it!
